docs/typography-page.html

```diff
@@ -128,39 +139,184 @@ <h4 class="sub-title gap-d20">COMPONENTS</h4>
             </div>
         </section>
     </aside>
+
+    <!-- Main Content -->
     <main class="content">
         <h1 class="txt-xlg txt-bold gap-d20">
```

Comment:
what is d20

Reply:
d-20 is a utility class i have that it stands for gap-down-20pixels